observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,6 +12,164 @@ You can use the {TempoName} xref:../otel/otel-forwarding-telemetry-data.adoc#ote
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+// git rebase marker 1
+
+[id="distr-tracing_3-4_{context}"]
+== Release notes for {DTProductName} 3.4
+
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+[id="distr-tracing_3-4_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-202?-????/[CVE-202?-????]
```

Comment:
```suggestion
* link:https://access.redhat.com/security/cve/CVE-2024-43796[CVE-2024-43796]
* link:https://access.redhat.com/security/cve/CVE-2024-43799[CVE-2024-43799]
* link:https://access.redhat.com/security/cve/CVE-2024-43800[CVE-2024-43800]
* link:https://access.redhat.com/security/cve/CVE-2024-45296[CVE-2024-45296]
* link:https://access.redhat.com/security/cve/CVE-2024-45590[CVE-2024-45590]
* link:https://access.redhat.com/security/cve/CVE-2024-45811[CVE-2024-45811]
* link:https://access.redhat.com/security/cve/CVE-2024-45812[CVE-2024-45812]
* link:https://access.redhat.com/security/cve/CVE-2024-47068[CVE-2024-47068]
* link:https://bugzilla.redhat.com/show_bug.cgi?id=2312579[npm-serialize-javascript: Cross-site Scripting (XSS) in serialize-javascript]
```

I don't know why some links don't work though... :confused: 

For example CVE-2024-45296 is also linked from https://access.redhat.com/errata/RHSA-2024:8014 (errata of another product).

Reply:
TY. Applied.